### PR TITLE
Changing template pages

### PR DESCRIPTION
**Pull request by AlexanderHodicke**

### CODE REVIEW

File: application-development/application-integration-and-process-automation/oracle-integration-cloud/README.md [hunk 1]
```diff
@@ -2,25 +2,32 @@
 
 *Product description*
 
+# Table of Contents
+
+1. [Team Publications](#team-publications)
+2. [Useful Links](#useful-uinks)
```
Comment (oheimburger) [inline application-development/application-integration-and-process-automation/oracle-integration-cloud/README.md:8]: Typo. Should read `#useful-links`.

Comment (AlexanderHodicke) [reply application-development/application-integration-and-process-automation/oracle-integration-cloud/README.md]: Done

**Comment by oheimburger** [comment]
Approved

**Comment by oheimburger** [comment]
Approved.

**Review by oheimburger — APPROVED** [review]

**Pull request merged**